lsp-metals.el

```diff
@@ -117,7 +117,9 @@ more customizations like using environment variables."
   (erase-buffer)
   (insert html)
   (shr-render-region (point-min) (point-max))
-  (setq-local buffer-read-only t))
+  (beginning-of-buffer)
```

Comment:
This function is for interactive use only;
  in Lisp code use `(goto-char (point-min))' instead.